public/assets/js/styleguide.js

```diff
@@ -72,4 +72,12 @@ document.addEventListener('DOMContentLoaded', function() {
       showDoc.checked = false;
   });
 
+  var reset = document.getElementById('reset');
+  reset.addEventListener('click', function() {
+    document.getElementById('documentation-button').checked = false;
+    document.getElementById('challenge-button').checked = false;
+    editor.setValue("");
+    ajax();
```

Comment:
It is to clear the editor, iframe and also the divs of challenges and documentation in the case of it is display.